UPGRADE-3.0.md

```diff
@@ -13,6 +13,7 @@
 # Backwards compatibility breaking changes
 
 *   Minimal PHP version was increased from v5.6.19 to v7.2.0.
+*   Minimal MySql version was increased from v5.5.3 to v5.7.14
```

Comment:
```suggestion
*   Minimal MySQL version was increased from v5.5.3 to v5.7.14
```
This is really a nitpick. But I think we should use the same spelling in all of the documents (in README.md we spell it as `MySQL`).